app/user/profile/ProfilePageLayout.tsx

```diff
@@ -0,0 +1,25 @@
+"use client"
+
+import React from "react"
+
+import DebuggingSession from "@/app/user/profile/DebuggingSession"
+import LoanList from "@/app/user/profile/LoanList"
+import LogoutButton from "@/app/user/profile/LogoutButton"
+
+const ProfilePageLayout = () => {
+  return (
+    <div className="content-container grid-go -mt-space-y w-full space-y-3">
+      <div className="col-span-full flex flex-row flex-wrap">
+        <h1 className="text-typo-subtitle-sm mb-5 lg:w-full">Profile</h1>
+        <LogoutButton className="ml-auto justify-end lg:order-2" />
+        <p className="text-typo-heading-2 mt-6 w-full pb-5 lg:order-1 lg:mt-0 lg:w-auto lg:max-w-[80%]">
+          Username / User Name
+        </p>
+      </div>
+      <LoanList />
+      <DebuggingSession />
```

Comment:
This should be removed before going into main 

```suggestion
      <DebuggingSession />
```

Reply:
It's been there since forever ago, and I think we should keep it until we implement the username. Me and Mikkel were talking yesterday and he said we should use the session for getting the username. So let's keep it until then.